Documentation/k8s-pre-reqs.md

```diff
@@ -20,6 +20,10 @@ Rook requires privileges to manage the storage in your cluster. See the details
 The Rook agent requires setup as a Flex volume plugin to manage the storage attachments in your cluster.
 See the [Flex Volume Configuration](flexvolume.md) topic to configure your Kubernetes deployment to load the Rook volume plugin.
 
+## Kernel with RBD module
+
+Rook requires a Linux Kernal built with the RBD module. Many distributions of Linux have this module but some don't, e.g. the GKE Container-Optimised OS (COS) does not have RBD. You can test your Kuberetes nodes by running `modprobe rbd`. If it says 'not found', you may have to [rebuild your kernel](https://rook.io/docs/rook/master/common-issues.html#rook-agent-rbd-module-missing-error) or choose a different Linux distribution.
```

Comment:
The Rook Ceph (operator) requires the RBD module, but not the other Rook operators, e.g., Cassandra, NFS.

@travisn @jbw976 do you think a separate section on this page is enough for each storage backend or a page per storage for the prerequisites would be better?

Reply:
@galexrt @whereisaaron The k8s prereqs page and all its subpages are really currently geared towards Ceph. There is hardly anything that applies to other providers that I can see. What if we rename the page in the TOC and the title to "Ceph Prerequisites"?

Reply:
I updated it to say "Rook Ceph requires..."

The prereqs page is [referenced by](https://github.com/rook/rook/search?q=k8s-pre-reqs.md&type=Code) many Rook modules, including Cassandra and NFS. So it is a grab-bag of requirements at the moment. Maybe you could reorganize it is into common and module-specific items. I just wanted the info to be present somewhere.